### PR TITLE
Xmxhelper

### PR DESCRIPTION
**Pull request by erlingjahr**
Added helper-method to calculate limitvalues from xmxvalue

### CODE REVIEW

File: charts/fint-altinn-ebevis-service/templates/deployment.yaml [hunk 1]
```diff
@@ -28,10 +28,10 @@ spec:
               mountPath: {{ .Values.deployment.volumeMounts.mountPath}}
           resources:
             limits:
-              memory: {{ .Values.deployment.resources.limits.memory}}
+              memory: "{{ include "fint-altinn-ebevis-service.xmx" . }}Mi"
```
Comment (mmedalen) [inline charts/fint-altinn-ebevis-service/templates/deployment.yaml]: change from "{{ include "fint-altinn-ebevis-service.xmx" . }}Mi" to {{ include "fint-altinn-ebevis-service.xmx" . }}Mi

File: charts/fint-altinn-ebevis-service/templates/deployment.yaml [hunk 2]
```diff
@@ -28,10 +28,10 @@ spec:
...
               cpu: {{ .Values.deployment.resources.limits.cpu}}
-            requests:
-              memory: {{ .Values.deployment.resources.requests.memory}}
+            requests: 
+              memory: "{{ div (include "fint-altinn-ebevis-service.xmx" . ) 2 }}Mi"
```
Comment (mmedalen) [inline charts/fint-altinn-ebevis-service/templates/deployment.yaml]: change from "{{ div (include "fint-altinn-ebevis-service.xmx" . ) 2 }}Mi" to {{ div (include "fint-altinn-ebevis-service.xmx" . ) 2 }}Mi

File: charts/fint-altinn-ebevis-service/values.yaml [hunk 3]
```diff
@@ -3,6 +3,7 @@
 # Declare variables to be passed into your templates.
 
 nameOverride: "altinn-ebevis-service"
+xmx: 644
```
Comment (mmedalen) [inline charts/fint-altinn-ebevis-service/values.yaml:6]: is this the default value we will set for all applications?

**Review by mmedalen — CHANGES_REQUESTED** [review]
would perhaps document the formula between xmx and memory in a read.md

File: charts/fint-altinn-ebevis-service/values.yaml (hunk 3 above)
Comment (erlingjahr) [reply charts/fint-altinn-ebevis-service/values.yaml:6]: No. This is for "altinn-ebevis-service"

**Pull request merged**